src/components/shortcodes/syntaxDiagram.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl'
+import React from 'react'
+import { Button, ButtonGroup, Grid } from '@material-ui/core'
```

Comment:
I recommend not to use `@material-ui/core` because our UI/UX designers want to use the `@material-ui/icons`. 😢

So we need to add `@material-ui/core` in our deps.

This is a force majeure factor. At the beginning of the development, we won't use any material design related. We choose [Bulma](https://bulma.io/) as the main framework.

I can help to transform to Bulma based code. If needed.